routes/cube_routes.js

```diff
@@ -885,8 +857,8 @@ router.get('/samplepack/:id/:seed', function(req, res) {
           metadata: generateMeta(
             'Cube Cobra Sample Pack',
             `A sample pack from ${cube.name}`,
-            `https://cubecobra.com/cube/samplepackimage/${req.params.id}/${pack.seed}`,
-            `https://cubecobra.com/cube/samplepack/${req.params.id}/${pack.seed}`
+            `http://162.243.163.51/cube/samplepackimage/${req.params.id}/${pack.seed}.png`,
```

Comment:
I assume you don't want this in production?

Reply:
Yes, just doing some tests on dev server. PR is ready for review now